overlay/files/system/oem/11_persistency.yaml

```diff
@@ -7,7 +7,7 @@ stages:
       environment:
         VOLUMES: "LABEL=COS_OEM:/oem LABEL=COS_PERSISTENT:/usr/local"
         OVERLAY: "tmpfs:25%"
-        RW_PATHS: "/var /etc /srv"
+        RW_PATHS: "/var /etc /srv /tmp"
```

Comment:
This seems quite off, `/tmp` should be mounted via systemd. Is there another underlying issue maybe?

Reply:
maybe tmp.mount is missing?